src/main/java/com/oracle/truffle/bf/test/BFRootNode.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.oracle.truffle.bf.test;
+
+import com.oracle.truffle.api.frame.VirtualFrame;
+import com.oracle.truffle.api.instrumentation.Instrumentable;
+import com.oracle.truffle.api.instrumentation.StandardTags;
+import com.oracle.truffle.api.nodes.ExplodeLoop;
+import com.oracle.truffle.api.nodes.Node;
+import com.oracle.truffle.api.nodes.RootNode;
+import com.oracle.truffle.api.source.SourceSection;
+
+@Instrumentable(factory = BFRootNodeWrapper.class)
```

Comment:
RootNode classes cannot be instrumented, because they cannot be replaced. Should actually produce an error, as far as I implemented that. If not we have found an issue.